### ISSUE TITLE
Ensure kind operator e2e test uses the same built images as our other tests

### ISSUE THREAD

**Issue by dprotaso** [knative-sandbox/net-contour#504]
Yeah, the separated line as below works with `ko` but now it hits contour-operator's issue https://github.com/projectcontour/contour-operator/issues/363

```
        - --contour-image 
        - ko://github.com/projectcontour/contour/cmd/contour
```

In short, contour-operator creates label from image tag, but `ko` command creates 64 length tag which exceeds labels length limiation.

_Originally posted by @nak3 in https://github.com/knative-sandbox/net-contour/issues/499#issuecomment-844675891_

**Comment by dprotaso** [comment]
While our other kind e2e tests are build from source in our vendor folder the operator currently does not

**Comment by nak3** [comment]
This should be able to fix by Contour 1.16 https://github.com/projectcontour/contour-operator/commit/f19b8b6333dd70b324e0d997ed7ba3c320c9be8c
